app/controllers/matches_controller.rb

```diff
@@ -1,6 +1,9 @@
 class MatchesController < ApplicationController
```

Comment:
This one has the same changes as the other branch: `scrapping-indeed` which one should be merged?